### ISSUE TITLE
sortDirection meaning swapped in version 59

### ISSUE THREAD

**Issue by vparusheva** [gregnb/mui-datatables#565]
I have a column with options for sortDirection, i.e.

```
const columns = [
      ...,
      {
        name: 'Updated',
        options: {
          sortDirection: 'desc',
        }
      },
     ...
];
```
Up until version 58 this worked fine.
However, since version 59, this sorts the items in ascending order. If I set the value to `asc`, they are sorted in descending order.

## Expected Behavior
'desc' must sort items in descending order (i.e. 'c', then 'b', then 'a')
'asc' must sort items in ascending order (i.e. 'a', then 'b', then 'c')

## Current Behavior
'desc' sorts items in ascending order
'asc' sorts items in descending order

## My Environment

| Tech         | Version |
|--------------|---------|
| Material-UI  |   3.9.2      |
| MUI-datatables  |   mui-datatables@2.0.0-beta.59    |
| React        |     16.7.0    |

**Comment by gabrielliwerant** [comment]
After the order got switched here https://github.com/gregnb/mui-datatables/commit/5e6064b2ca524ff175813bc7e960924010997e9e, it needed to also be switched here https://github.com/gregnb/mui-datatables/blob/master/src/utils.js#L24.

I can fix if you want.

**Comment by gabrielliwerant** [comment]
PR open here https://github.com/gregnb/mui-datatables/pull/570

**Comment by gregnb** [comment]
Should be resolved upgrade to the latest version 2.0.0
